src/components/HotspotListItem.tsx

```diff
@@ -68,6 +78,14 @@ const HotspotListItem = ({
     return `${geo.longStreet}, ${geo.longCity}, ${geo.shortCountry}`
   }, [hotspot, t])
 
+  const isRelayed = useMemo(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    // TODO: fix listen_addrs in helium-js for witnesses
+    const listenAddrs = hotspot?.status?.listen_addrs
```

Comment:
Looks like the witness hotspot objects dont have camel case for `listen_addrs`

Reply:
Should we fix helium-js before merging?

Reply:
I might need to make another change there for explorer, but was planning to merge and fix it later just to get this out since the release is waiting for it.